packages/qgrid-core/src/command/command.js

```diff
@@ -1,14 +1,14 @@
-import { yes } from '../utility/kit';
 import { SubjectLike } from '../rx/rx';
+import { yes } from '../utility/kit';
 
 export class Command {
-	constructor(context = {}) {
+	constructor(context = {}, priority = 1) {
```

Comment:
let's keep priority -1 as a default (it will mean no priority), put prioirty under context please